.github/workflows/CI_push_pypi.yml

```
@@ -0,0 +1,40 @@
+name: Pypi Build
+
+on: [push]
+
+jobs:
+
+  build:
+    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        python-version: ['3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python
+          python --version
```

For MacOS failures, some OS have symlink of python -> python2 other OSes have python -> python3
```suggestion
          python3 --version
```

or maybe removing shell will help it pick right python

---

Thanks Stefan! I removed the ``shell`` part.